CONTRIBUTING.rst

```diff
@@ -60,6 +60,33 @@ Once you have done this, you can launch the master branch of Jupyter notebook
 from any directory in your system with::
 
     jupyter notebook
+  
+Verification 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
While yo are fixing what's just below can you have the same number of `^` as the lenght of the title ?

Thanks !